pkg/p2p/kadcast/packet.go

```diff
@@ -11,6 +11,13 @@ type Packet struct {
 	payload []byte
 }
 
+func makePacket(headers [24]byte, payload []byte) Packet {
```

Comment:
It is more idiomatic to name constructor-like functions with `new` prefix instead of `make`

Reply:
I red on the book that this is the way for naming functions in go which only purpose is be a constructor and space allocator for an Struct.

But will change all of them if it's wrong since all of the constructors have the `make` instead of `new`.